File: src/main/java/com/example/Project/Ticket/TicketResponse.java
package com.example.Project.Ticket;

import lombok.Builder;
import lombok.Data;

import java.time.LocalDate;

@Data
@Builder
public class TicketResponse {
    private int id;
    private String subject;
    private String username;
    private LocalDate date;
    private Ticket.Type type;
    private Ticket.Status status;
    private Ticket.Priority priority;

}

File: src/app/models/ticket-response.ts
/* tslint:disable */
/* eslint-disable */
// ticket-response.model.ts
import { TicketPriority, TicketType, TicketStatus } from '../User/dashbord/dashbord.component'; 
export interface TicketResponse {
  id: number;
  subject: string;
  username: string;
  type: TicketType;
  date: Date;
  status: TicketStatus;
  priority: TicketPriority;
  locked: boolean; 
}
  